static/src/javascripts/projects/common/modules/commercial/templates/acquisitions-banner-fiv.js

```diff
@@ -0,0 +1,46 @@
+// @flow
+import marque36icon from 'svgs/icon/marque-36.svg';
+import closeCentralIcon from 'svgs/icon/close-central.svg';
+import arrowWhiteRight from 'svgs/icon/arrow-white-right.svg';
+import config from 'lib/config';
+
+export const acquisitionsBannerFivTemplate = (
+    location: string,
+    linkUrl: string,
+): string =>
+    `
+    <div class="fiv-banner__close">
```

Comment:
this div element is a copy of the membership one, will be edited with the proper design.